optuna/visualization/_imports.py

```diff
@@ -0,0 +1,11 @@
+try:
```

Comment:
IMO, the name "_imports" is too generic. How about renaming it to "_plotly_imports" (or something else)?

Reply:
Besides, I'm not sure that we should add `_` prefix to this module because other private modules (e.g., `optuna/samplers/tpe/parzen_estimator.py`) don't have the prefix.
I want to hear opinions from other dev members.

Reply:
I renamed this to `plotly_imports`.